src/libraries/System.Configuration.ConfigurationManager/src/System/Configuration/ClientConfigPaths.cs

```diff
@@ -265,9 +264,12 @@ private static string GetTypeAndHashSuffix(string exePath, bool isSingleFile)
                 {
                     try
                     {
-                        // Certain platforms may not have support for crypto
-                        hash = IdentityHelper.GetNormalizedUriHash(codeBase);
-                        typeName = UrlDesc;
+                        if (Uri.TryCreate(Path.Combine(AppDomain.CurrentDomain.BaseDirectory, assembly.ManifestModule.Name), UriKind.RelativeOrAbsolute, out Uri codeBase))
```

Comment:
Do we have any test coverage for this? For example, you delete this code completely - are any tests going to fail?

Reply:
Doesn't seem there is a test, no test failure if delete it completely

Reply:
Are you suggesting to remove this part or add a test (not sure if it is testable though)?